### ISSUE TITLE
Overdue bounties display incorrect information

### ISSUE THREAD

**Issue by javieralaves** [AutarkLabs/open-enterprise#1797]
**Describe the bug**
Bounties that are overdue continue to accept applicants and work submissions as if the deadline termination had no effect at all.

**Mainnet or testnet?**
Rinkeby

**Organization**
[Continuous deploy DAO](https://rinkeby.aragon.org/#/neudune/0xe3fc72a7f3dda98a0a76863c9aad73968c6c0c42/)

**To Reproduce**
1. Go to Projects
2. Go to Issues tab
3. Notice there are overdue bounties that display incorrect information
4. Click into one of them
5. Notice that the bounty widget displays incorrect information and the context menu as well

**Expected behavior**
I would expect the information in the Issues tab to reflect the new bounty status, and remove the difficulty and bounty amount, as these are unnecessary. New funding should be allowed from the context menu. [(specific reference here)](https://www.figma.com/file/Xnn4lQJX4d5AautuvmAlnxHf/Projects?node-id=1364%3A90)

In the issue detail view, I would expect the bounty widget to reflect the correct information, that is: 
* Remove the bounty amount
* Keep the deadline, number of bounties, and applications/work submissions, as long as they can't be interacted with from their respective panel views [(specific reference here)](https://www.figma.com/file/Xnn4lQJX4d5AautuvmAlnxHf/Projects?node-id=1364%3A90)

**Screenshots**
Currently on the continuous deploy DAO:
<img width="1440" alt="Screenshot 2019-12-18 at 09 07 46" src="https://user-images.githubusercontent.com/28843778/71069764-a4ed5c80-2179-11ea-809d-6489d46a1802.png">
<img width="1440" alt="Screenshot 2019-12-18 at 09 07 55" src="https://user-images.githubusercontent.com/28843778/71069795-b9c9f000-2179-11ea-9d94-84b83f1fc588.png">

**Comment by stellarmagnet** [comment]
@javieralaves just thinking about this more, and pondering this:

> I would expect the information in the Issues tab to reflect the new bounty status, and remove the difficulty and bounty amount, as these are unnecessary.

I think that we should show what the previous details for the bounty in the "activity" panel, otherwise it can be confusing to users on why a bounty disappeared.

E.g.

"This issue was allocated a bounty of 50 ANT and beginner difficulty. The deadline has been reached today without any accepted work. You can [fund this issue again](link)."

Where the link would open the Fund Issue panel

cc @rkzel

**Comment by javieralaves** [comment]
@stellarmagnet Agreed that we should keep reference information on the Activity panel. So far, we planned for the bounty widget to display the following information:

<img width="357" alt="No applications accepted" src="https://user-images.githubusercontent.com/28843778/71961888-a3eba000-31f8-11ea-95c2-a5db535f152b.png">

My main concern in this ticket was that applications were still being accepted even though the deadline passed and the bounties were no longer available.

What do you think about adding that information to the Bounty info panel instead?

<img width="357" alt="No applications accepted - v2" src="https://user-images.githubusercontent.com/28843778/71962632-232da380-31fa-11ea-8b80-b8c86bf07d8b.png">

Alternatively, I think a status update on the Activity panel is a good idea worth a new ticket.
